src/integration/integration_golangapi_eventgenerator_test.go

```diff
@@ -259,7 +259,99 @@ var _ = Describe("Integration_GolangApi_EventGenerator", func() {
 				By("check public api")
 				checkAggregatedMetricResult(components.Ports[GolangAPIServer], pathVariables, parameters, result)
 			})
+			It("should get the metrics in specified time scope", func() {
+				By("get the results from 555555")
+				parameters = map[string]string{"start-time": "555555", "order-direction": "asc", "page": "1", "results-per-page": "10"}
+				result := AppAggregatedMetricResult{
+					TotalResults: 3,
+					TotalPages:   1,
+					Page:         1,
+					Resources: []models.AppMetric{
+						models.AppMetric{
+							AppId:      appId,
+							MetricType: models.MetricNameMemoryUsed,
+							Unit:       models.UnitMegaBytes,
+							Value:      "123456",
+							Timestamp:  555555,
+						},
+						models.AppMetric{
+							AppId:      appId,
+							MetricType: models.MetricNameMemoryUsed,
+							Unit:       models.UnitMegaBytes,
+							Value:      "123456",
+							Timestamp:  555555,
+						},
+						models.AppMetric{
+							AppId:      appId,
+							MetricType: models.MetricNameMemoryUsed,
+							Unit:       models.UnitMegaBytes,
+							Value:      "123456",
+							Timestamp:  666666,
+						},
+					},
+				}
+				By("check public api")
```

Comment:
please remove all `By("check public api")` as in golangapiserver integration tests there is no internal api.